### ISSUE TITLE
hello, happen error: raise NoBackendError()

### ISSUE THREAD

**Issue by qqingdou** [giulbia/baby_cry_detection#2]
when i run the train_set.py, the error happen bellow:
data, sample_rate = file_reader.read_audio_file()
  File "/usr/lib/python2.7/site-packages/baby_cry_detection-1.0-py2.7.egg/pc_methods/__init__.py", line 35, in read_audio_file
    audio_data, sr = librosa.load(self.file_name, sr=44100, mono=True, duration=5)
  File "/usr/lib/python2.7/site-packages/librosa-0.5.0-py2.7.egg/librosa/core/audio.py", line 107, in load
    with audioread.audio_open(os.path.realpath(path)) as input_file:
  File "/usr/lib/python2.7/site-packages/audioread/__init__.py", line 115, in audio_open
    raise NoBackendError()
audioread.NoBackendError

**Comment by giulbia** [comment]
Hi,

If I rember correctly, I had the same error once at reading files. The problem was a hidden file (.DS_store, l'équivalent Mac OS X du Thumbs.db on Windows) in the folder where I was attempting to read audio files. Removing it should solve the problem.

**Comment by qqingdou** [comment]
Hi,  I  am not find this file named .DS_store on Linux System..

**Comment by giulbia** [comment]
With `ls -la` you can find out if there's anything that is not an audio file in the folder and remove it.

**Comment by qqingdou** [comment]
i used ls -la command to search file in the folder, there is nothing except ogg files list.
it may be other error, let me try again and solve it, thanks

**Comment by qqingdou** [comment]
Hi, i have resolved the error "audioread.NoBackendError", but another error happen bellow:
Traceback (most recent call last):
  File "train_set.py", line 83, in <module>
    main()
  File "train_set.py", line 63, in main
    avg_features = feature_engineer.feature_engineer(audio_data=data)
  File "/usr/lib/python2.7/site-packages/baby_cry_detection-1.0-py2.7.egg/pc_methods/feature_engineer.py", line 112, in feature_engineer
    ), axis=0)
ValueError: all the input array dimensions except for the concatenation axis must match exactly

**Comment by giulbia** [comment]
So what was the problem for the first error ?

Which commit have you downloaded? I've made modifications, I can't say what's wrong like that.

**Comment by qqingdou** [comment]
I have installed ffmpeg,  So the first error "audioread.NoBackendError" disappeared.

When i  run “python train_set.py" again, the error happened below:
ValueError: all the input array dimensions except for the concatenation axis must match exactly

**Comment by giulbia** [comment]
I think is the same issue someone else is having. Which commit have you cloned ?

**Comment by yajuusennpai** [comment]
same problem, solved by running "conda install ffmpeg -c conda-forge"
